packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -63,12 +63,13 @@ const MainContent = ( {
 				'You have not yet created any menus. Displaying a list of your Pages'
 		  );
 	return (
-		<OffCanvasEditor
+		<PrivateListView
 			blocks={ clientIdsTree }
-			parentClientId={ clientId }
+			parentId={ clientId }
 			isExpanded={ true }
-			LeafMoreMenu={ LeafMoreMenu }
 			description={ description }
+			showAppender={ true }
```

Comment:
Not introduced here, but the boolean props could be shorted to just `showAppender` to save a few bytes.